src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+
+-- {{{ console configuration
+
+M['console'] = format_text([[
+    Configure the administrative console. A client to the console is `tt`
+    connect.
+]])
+
+M['console.enabled'] = format_text([[
+    Whether to listen on the Unix socket provided in the console.socket option.
+
+    If the option is set to `false`, the administrative console is disabled.
+]])
+
+M['console.socket'] = format_text([[
+    The Unix socket for the administrative console.
+
+    Mind the following nuances:
+
+    1. Only a Unix domain socket is allowed. A TCP socket can't be configured
+    this way.
+
+    2.console.socket is a file path, without any `unix:` or `unix/:` prefixes.
+
+    3. If the file path is a relative path, it is interpreted relative
+    to `process.work_dir`.
+]])
+
+-- }}} console configuration
+
+-- {{{ credentials configuration
+
+M['credentials'] = format_text([[
+    The `credentials` section allows you to create users and grant them the
+    specified privileges.
+]])
+
+-- {{{ credentials.roles configuration
+
+M['credentials.roles'] = format_text([[
+    An array of roles that can be granted to users or other roles.
+]])
+
+M['credentials.roles.*'] = 'Role name.'
+
+M['credentials.roles.*.privileges'] = format_text([[
+    An array of privileges granted to this role.
+]])
+
+M['credentials.roles.*.privileges.*'] = format_text([[
+    Privileges that can be granted to a user with this role.
+]])
+
+M['credentials.roles.*.privileges.*.functions'] = format_text([[
+    Functions to which user with this role gets the specified permissions.
+]])
+
+M['credentials.roles.*.privileges.*.functions.*'] = 'Function name.'
+
+M['credentials.roles.*.privileges.*.lua_call'] = format_text([[
+    Defines the Lua functions that the user with this role has permission to
+    call. This field accepts a special value, `all`, which grants the privilege
+    to use any global non-built-in Lua functions.
+]])
+
+M['credentials.roles.*.privileges.*.lua_call.*'] = 'Lua function name.'
+
+M['credentials.roles.*.privileges.*.lua_eval'] = format_text([[
+    Whether this user with this role can execute arbitrary Lua code.
+]])
+
+M['credentials.roles.*.privileges.*.permissions'] = format_text([[
+    Permissions assigned to user with this role.
+]])
+
+M['credentials.roles.*.privileges.*.permissions.*'] = 'Permission name.'
+
+M['credentials.roles.*.privileges.*.sequences'] = format_text([[
+    Sequences to which user with this role gets the specified permissions.
+]])
+
+M['credentials.roles.*.privileges.*.sequences.*'] = 'Sequence name.'
+
+M['credentials.roles.*.privileges.*.spaces'] = format_text([[
+    Spaces to which user with this role gets the specified permissions.
+]])
+
+M['credentials.roles.*.privileges.*.spaces.*'] = 'Space name.'
+
+M['credentials.roles.*.privileges.*.sql'] = format_text([[
+    Whether user with this role can execute an arbitrary SQL expression.
+]])
+M['credentials.roles.*.privileges.*.sql.*'] = 'SQL expression name.'
```

Comment:
Only `all` is allowed for now, it worth to note IMHO.

Reply:
Added.